ocs_ci/ocs/machine.py

```diff
@@ -0,0 +1,97 @@
+import re
+import logging
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.ocs import OCS
+
+
+log = logging.getLogger(__name__)
+
+
+def get_machine_objs(machine_names=None):
+    """
+    Get machine objects by machine names
+
+    Args:
+        machine_names (list): The machine names to get their objects for.
+                              If None, will return all cluster machines
+
+    Returns:
+        list: Cluster machine OCP objects
+    """
+    machines_obj = OCP(kind='machine', namespace='openshift-machine-api')
```

Comment:
openshift-machine-api can be added to constant

Reply:
Done